pages/doc.js

```diff
@@ -14,14 +14,15 @@ import Hamburger from '../src/Hamburger'
 import fetch from 'isomorphic-fetch'
 import kebabCase from 'lodash.kebabcase'
 import compact from 'lodash.compact'
-import flatten from 'lodash.flatten'
 import { scroller, animateScroll } from 'react-scroll'
 import 'core-js/fn/array/find-index'
 // styles
 import styled from 'styled-components'
 import { media } from '../src/styles'
 // json
 import sidebar from '../src/Documentation/sidebar'
+import SidebarMenuHelper from '../src/Documentation/SidebarMenu/SidebarMenuHelper'
+import { PATH_SEPARATOR } from '../src/Documentation/SidebarMenu/SidebarMenuHelper'
```

Comment:
looks a bit too much, it's always '/' - it's easier to use just plain '/' everywhere.

Reply:
fix it